components/SpotlightBanner/SpotlightBannerAside.tsx

```diff
@@ -1,7 +1,11 @@
 import React, { FunctionComponent } from 'react';
 
-const SpotlightBannerAside: FunctionComponent = ({ children }) => (
-  <div className="spotlight-banner__aside">{children}</div>
+interface SpotlightBannerAsideProps {
+  classname: string;
+}
+
+const SpotlightBannerAside: FunctionComponent<SpotlightBannerAsideProps> = ({ children, classname }) => (
+  <div className={classname}>{children}</div>
```

Comment:
Is there a `SpotlightBannerAside` element that doesn't need the `spotlight-banner__aside` class? I don't this. So why remove it? The whole point of this is to abstract identical code while allowing for subtle changes e.g a SpotlightBannerAside should always have the `spotlight-banner__aside` class, but it can have other classes as well.